### ISSUE TITLE
Stop working after kernel upgrade from 4.20.7 to 4.20.10

### ISSUE THREAD

**Issue by RistiCore** [TelegramMessenger/MTProxy#270]
Server: VPS (KVM), with two IPv4 (one of them configured as alias and this alias used as incoming for mtproxy connections). 
Mtproxy starts with param `--nat-info=alias-ip:main-ip`.

On kernel 4.20.7 everything works fine. After upgrade to kernel 4.20.10 mtproxy broken. If I boot this server with previous (4.20.7) kernel everything works again.

Log file is very long, but comparing mtproxy logs for 4.20.7 and 4.20.10 I cannot see any signigicant difference of errors in 4.20.10 case.

**Comment by uprt** [comment]
What do you mean exactly as "stop working"? It doesn't start? It doesn't handle incoming connections? Or what?

Have you tried to set logging verbosity level to 2 or 3 (by adding an option '-v 2' or '-v 3' to the command line) when comparing the logs?

**Comment by RistiCore** [comment]
Daemon succ starts, no any errors in log, but there are rapid activity of connection-disconnection. I'll attach log samples.
After downgrade to 4.20.7 everything works fine.
On systems with single IPv4 (no --nat-info option in mtproto-proxy start) everything works fine on 4.20.10.

**Comment by daskol** [comment]
I have the same issue with infinite connection-disconnection loop on kernel version 4.4.0.

**Comment by RistiCore** [comment]
> I have the same issue with infinite connection-disconnection loop on kernel version 4.4.0.

After the last kernel update?

I have two proxy servers, each on OpenSUSE with kernel 4.20.10.
One server with `--nat-info` option has infinite connection loop. Another service without this option works fine.

**Comment by daskol** [comment]
> After the last kernel update?

No, I just run MTProxy on new VPS as it described in manual. I do not think that the issue related to kernel version since I have run MTProxy sucessfully on other VPS's.
```
$ /opt/MTProxy/mtproto-proxy -u nobody -p 8888 -H 8443 -S $SECRET --aes-pwd proxy-secret proxy-multi.conf -M 1
```

**Comment by RistiCore** [comment]
After update to `kernel-default-4.20.13` mtproto-proxy works fine again. With the same config files for service start.

**Comment by daskol** [comment]
@stek29 Why did you close the issue? Kernel version should not affect this kind of software if there are no dirty hacks in mtp-proxy itself. I belive that the bug is still here but not in kernel.

**Comment by RistiCore** [comment]
I can attach two log files after some anonymizing: one for 4.20.7 when everything is ok, and one for 4.20.10 when connections fail.

**Comment by RistiCore** [comment]
Logs from OK-working kernel 4.20.7: https://yadi.sk/d/pw-OimpjSptYaQ
Logs from NOK-working kernel 4.20.10: https://yadi.sk/d/EC88koMuwDO8Vg

**Comment by uprt** [comment]
Looks like this issues can have something in common:
https://github.com/TelegramMessenger/MTProxy/issues/50
https://github.com/TelegramMessenger/MTProxy/issues/132
https://github.com/TelegramMessenger/MTProxy/issues/70
